--- ReactNative.podspec
+++ ReactNative.podspec
@@ -19,6 +19,7 @@ Pod::Spec.new do |s|
   s.source_files = "ios/**/*.{h,m,mm,swift}", "ios/generated/**/*.{h,m,mm}", "cpp/**/*.{hpp,cpp,c,h}", "cpp/generated/**/*.{hpp,cpp,c,h}"
   s.vendored_frameworks = "FedimintReactNativeFramework.xcframework"
+  s.frameworks = "SystemConfiguration", "Security"
   s.dependency    "uniffi-bindgen-react-native", "0.29.3-1"